src/app/api/v1/retrofunding/rounds/[roundId]/ballots/[ballotAddressOrEns]/impactMetrics/[impactMetricId]/route.ts

```diff
@@ -0,0 +1,15 @@
+export async function DELETE(
+  request: Request,
+  route: {
+    params: {
+      roundId: string;
+      ballotAddressOrEns: string;
```

Comment:
Let's keep it consistent with other endpoints and call it `addressOrEnsName`. This is the address that is assigned to the user and not to the ballot itself

Reply:
cool, that clears up a question I had on #244 too